### DISCUSSION TITLE
1.0.6 업데이트 후 Create Custom Environment 로 접속하던게

### DISCUSSION THREAD

**Discussion by mhkim0720** [naverz/zepeto-studio-kor#44] [Issue Report (Legacy)]
안되네요 피시 두대로 테스트 하던 건데 업데이트 후 발생한 문제가 맞나요?

업데이트 전에 테스트 환경은 서버 역할 피시 를 Development 로 놓고
클라이언트 역할 피시에서 Custom environment 로 아이피 , 18080 으로 접속해서 테스트 했었습니다.

답변 부탁합니다. ~

**Comment by NoahHahm** [comment]
버그는 아니지만, 이번에 멀티프로젝트에서 포트점유 문제를 해결하고자 고정 포트 바인딩 정책이 폐지 되었습니다.
에디터(프로젝트) 처음 켰을때 사용자 PC 에서 가용가능한 포트를 기준으로 바인딩 되는데요.
PlayMode 시 인스펙터에서 해당 포트를 확인 하실수있고 (**에디터에서는 18080 표기되는 버그는 다음 버전에 수정** 됩니다.)
혹은 .vscode/launch.json 파일 안에 있는 Multiplay Port 포트를 확인 하실 수 있습니다.
Custom Environment 에서 해당 포트로 접근하시면 됩니다. (단 에디터를 다시 켰을때 포트가 변동될 수 있습니다.)

![image](https://user-images.githubusercontent.com/6405162/145367366-7d5b0501-072d-4487-a8d7-63342ea28346.png)
![image](https://user-images.githubusercontent.com/6405162/145367566-73e505a5-adaf-4248-ae71-3b164598e88e.png)

**Reply by mhkim0720** [reply]
답변 감사합니다. 말씀하신 대로 해봤는데 

클라이언트 역할 피시의 포트는 어떻게 설정하나요?  접속이 안됩니다. 

development / custom environment 두개 다 해봐도 안되서 launch.json 파일의 접속 포트를 저장해서 해도 안되네요 

답변 부탁합니다.

**Reply by NoahHahm** [reply]
위 파일은 수정하시면 안됩니다.
최초 프로젝트 로드후 해당 파일에 멀티플레이 게이트웨이 포트가 저장되어 vscode 로 디버깅을 위한 포트 입니다.

따라서 프로젝트 로드후 해당 포트를 기억하시고
custom env 만드신 환경에서 해당 포트를 기입해서 잡속하시면 됩니다

**Comment by mhkim0720** [comment]
자답 입니다.. 
공식 답변대로 해도 안되서 포기하다가 하이어라키상에 월드멀티플레이를 클릭하면 인스펙터에 나오는 포트 번호로 접속하면 
접속 됩니다. 이유는 모르겠으나 런타임에 저 포트가 launcher 의 포트와 다른 포트로 열리네요 
쨋든 해결~

**Comment by hyeonjin-jo** [comment]
안녕하세요

https://studio.zepeto.me/kr/guides/multiplay
위의 문서의 중간쯤에 ZEPETO Multiplay창에서 port번호 확인이 가능한 부분 업로드 하였습니다.

참고부탁드립니다.
